lib/grpperm.gd

```diff
@@ -528,6 +528,22 @@ DeclareGlobalFunction( "ReducedPermdegree" );
 
 DeclareGlobalFunction("MovedPointsPerms");
 
+#############################################################################
+##
+#F  LogPerm( <a>, <b> )
+##
+##  <ManSection>
+##  <Func Name="LogPerm" Arg='a,b'/>
+##
+##  <Description>
+##  For two permutations <A>a</A> and <A>b</A> this function returns a
+##  positive integer <A>e</A> such that <A>a^e=b</A>, or <K>false</A> if no such
+##  element exists.
+##  </Description>
+##  </ManSection>
```

Comment:
Just to observe that this is not documented (fine with me, just in case).

Also, would it be better to return `fail` instead of `false`?

Reply:
I'd prefer to stay with `false` for a valid "not a number" output, and keep `fail` for "calculation did not complete".